db/updates.go

```diff
@@ -140,6 +140,38 @@ func (d *Database) ProcessUpdateForUsage(ctx context.Context, update *Update) er
 		}
 		log.Debugf("after getting active user plan %s", subscription.ID)
```

Comment:
This line will have to be updated to omit the subscription ID from the message if it's not found. It might also be reasonable to move this line down past the check to ensure that the default subscription exists.

Reply:
I think that it might not actually be necessary. Looking at `GetActiveSubscription`, it only returns `nil` if it's returned an error. I can probably remove the check for nil below, which should also satisfy the linter, I think.